packages/visualizations/src/components/Table/types.ts

```diff
@@ -8,9 +8,17 @@ export type DataFormat = typeof DATA_FORMAT[DataFormatKey];
 
 export type TableData = DataFrame;
 
+export const ColumnSort = {
+    asc: 'ASC',
+    desc: 'DESC',
+} as const;
+export type ColumnSortValues = typeof ColumnSort[keyof typeof ColumnSort];
+
 type BaseColumn = {
     key: string;
     title: string;
+    sorted?: ColumnSortValues;
+    onClick?: () => void;
```

Comment:
If we have to handle mutli sort, how this could evolve ? 
Add another property like sort position or update `sorted` to a tuple like `['ASC', 0]` ? 

Maybe we could benefit to move the sort pros as a first level `options` prop where it's an array (to handle multi sort) where we store an object with the field key and sort order


```js
sort: [
   {key: 'fieldA', sort: 'ASC'},
   {key: 'fieldB', sort: 'DESC'},
   // ...
]
```

Reply:
As it is implemented, it's not a "sort" function, it's just a visual indicator that this column is sorted. What the outter app decides to do with it isn't SDK's problem imo. But whatever they do, the visual indication will remain the same I think.

Reply:
Yes, I agree we should keep the prop as a visual indicator on how the outer app has decided to represent data.

My point was to merge the sorted key for each column into a single prop at the root of the options prop to ease the future dev if we need to display a sort order indication between fields.

Reply:
Both could work I guess, but I'd rather keep columns enclosing and add a `sortPrority` key later, if we ever need it (which I doubt). The only occurence of a multisort I know of is airtable and the sort order is not in the table itself.

Reply:
Ok let's see how the product will drive this feature in the future